packages/core/src/next.ts

```diff
@@ -1,7 +1,3 @@
-export * from "./components/Heading/Heading";
-import Heading from "./components/Heading/Heading";
-export * from "./components/EditableHeading/EditableHeading";
-import EditableHeading from "./components/EditableHeading/EditableHeading";
-import Search from "./components/Search/Search";
-
-export { Heading, EditableHeading, Search };
+export * from "./components/Heading";
```

Comment:
I think you've once said that this is not the fils which decides which components are exported, I think it is something that was added only for the playground